src/libraries/CurrencyDelta.sol

```diff
@@ -15,21 +15,21 @@ library CurrencyDelta {
         }
     }
 
-    /// @notice sets a new currency delta for a given caller and currency
-    function setDelta(Currency currency, address caller, int256 delta) internal {
+    /// @notice applies a new currency delta for a given caller and currency
+    /// @return previous the prior value
+    /// @return next the modified result
+    function applyDelta(Currency currency, address caller, int128 delta)
```

Comment:
can we use `target` here as well instead of `caller`

Reply:
It was `caller` in the previous functions.

four remaining instances of `caller`:
```
src/libraries/CurrencyDelta.sol:10:    function _computeSlot(address caller_, Currency currency) internal pure returns (bytes32 hashSlot) {
src/libraries/CurrencyDelta.sol:21:    function applyDelta(Currency currency, address caller, int128 delta)
src/libraries/TransientStateLibrary.sol:44:    function currencyDelta(IPoolManager manager, address caller_, Currency currency) internal view returns (int256) {
```
two instances of `target`:
```
src/PoolManager.sol:323:    function _accountDelta(Currency currency, int128 delta, address target) internal {
src/PoolManager.sol:336:    function _accountPoolBalanceDelta(PoolKey memory key, BalanceDelta delta, address target) internal {
```

I think `caller` is more descriptive since the targets can only be `msg.sender` which is the `CALLER` opcode. I'd rather make everything `caller`.

Also, if `caller` is always `msg.sender`, the parameter could be removed entirely to save slightly more gas. However, I suspect some of the tests may specify a custom caller for these functions.

After you acknowledge this and insist one way or the other, I will make whatever change you recommend.

Reply:
Its not always msg.sender

If you look in `modifyLiquidity` and `swap` sometimes deltas are accrued to hooks

Reply:
Then I'll change all three instances of `caller` to `target`.

Reply:
yeah agreed with sara that `target` makes more sense given that hooks are given deltas sometimes too :) good catch